integrations-service/integrations/autogen/Tasks.py

```diff
@@ -481,43 +779,131 @@ class ImageUrl(BaseModel):
     """
 
 
-class LogStep(BaseModel):
+class LogStep(BaseModel):
+    model_config = ConfigDict(
+        populate_by_name=True,
+    )
+    kind_: Annotated[Literal["log"], Field(json_schema_extra={"readOnly": True})] = "log"
+    """
+    The kind of step
+    """
+    label: str | None = None
+    """
+    The label of this step for referencing it from other steps
+    """
+    log: str
+    """
+    The value to log
+    """
+
+
+class Main(BaseModel):
```

Comment:
There's something wrong with this file in general. This is the `map` step. And seems like the old `Main` model is called `MainModel` now.